src/experimental/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * ALL SYMBOLS EXPOSED FROM THIS MODULE ARE EXPERIMENTAL AND MIGHT
+ * BE SUBJECT OF BREAKING CHANGES WITHOUT NOTICE OR LIBRARY MAJOR VERSION CHANGE.
+ */
+
+// Types
+export * from './types';
+
+// Components
+
+export { default as ScreenStackHost } from '../components/gamma/ScreenStackHost';
+export {
+  default as StackScreen,
+  StackScreenLifecycleState,
```

Comment:
(non-blocking) it should be moved to types, but keeping in mind that we should extract types for the new stack like we did for other components, it's fine to just create a ticket for covering `StackScreen.types`

Reply:
ack